core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -17,32 +17,27 @@
 
 package kafka.admin
 
-import java.io.PrintStream
```

Comment:
I'd like this order optimization. However, it is not related to this PR and it may produce conflicting files to other PRs. Hence, could you please revert this change?